### ISSUE TITLE
GWS.GMAIL.23 Feedback

### ISSUE THREAD

**Issue by amart241** [cisagov/ScubaGoggles#62]
TLDR: I recommend cutting GWS.GMAIL.23. See below for my reasoning.

## GWS.GMAIL.23.1v0.1
The policy currently reads as "Attachment compliance SHOULD be enabled to filter specific attachments within Gmail messages." However, attachment compliance is not a simple on/off setting, so saying it should be enabled doesn't really mean anything. Even after reading the implementation, it's unclear what we actually are telling the users to do. Attachment compliance allows you to construct custom rules (similar to EXO's mail flow rules). For example, you could craft a rule such as ""if you have an incoming email from an external sender, prepend 'External: ' to the subject line." Saying to enable attachment compliance without specific details about what rules to craft is meaningless. Recommend cutting this control.

## GWS.GMAIL.23.2v0.1
It might be worth noting the limitations Gmail itself has with true file type detection. It can only perform this detection on a select few file types (see https://apps.google.com/supportwidget/articlehome?hl=en&article_url=https%3A%2F%2Fsupport.google.com%2Fa%2Fanswer%2F2364580%3Fhl%3Den&assistant_id=generic-unu&product_context=2364580&product_name=UnuFlow&trigger_context=a&fragment=file-types for more info). For all others it just matches by file extension. Given these limitations, I'm not sure if this a completely fair requirement.

## GWS.GMAIL.23.3v0.1
Which file types should be blocked? "Executable" files are blocked by default, and can't be unblocked (https://support.google.com/mail/answer/6590?fl=1&sjid=1223071020538845302-NC#zippy=%2Cmessages-that-have-attachments). This is not the case for M365, so I don't think this control makes as much sense here.

**Comment by adhilto** [comment]
NOTE: This control is note removed by #138. After that PR is merged into main, this issue now refers to GWS.GMAIL.19.1v0.1.

**Comment by jkaufman-mitre** [comment]
@adhilto We will discuss with the team and make a decision.

**Comment by jkaufman-mitre** [comment]
@adhilto We will remove the whole section from the baseline and defer to the hardening guide.

**Comment by jkaufman-mitre** [comment]
Changes were made and the PR was created.
